### PR TITLE
Adopt “Avoid recommending a local container image build” changes for zh

### PR DESCRIPTION
**Pull request by LJTian**
Sync #34112 Changes

### CODE REVIEW

**Comment by jihoon-seo** [comment]
@LJTian Please sign the CLA. (See [this](https://github.com/kubernetes/website/pull/34929#issuecomment-1180091089))
/language zh

**Comment by LJTian** [comment]
ok, in progress

File: README-zh.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ To build the site in a container, run the following to build the container image
 要在容器中构建网站，请通过以下命令来构建容器镜像并运行：
 
 ```bash
-make container-image
+# 您可以将 $CONTAINER_ENGINE 设置为任何 Docker 类容器工具的名称
```
Comment (tengqm) [inline README-zh.md:83]:
```suggestion
# 你可以将 $CONTAINER_ENGINE 设置为任何 Docker 类容器工具的名称
```

**Comment by sftim** [comment]
/retitle Adopt “Avoid recommending a local container image build” changes for zh

**Pull request merged**